src/feature-test/user_see_form.test.js

```diff
@@ -1,10 +1,35 @@
-import '../setupTests.js'
-import App from '../App';
+import '../setupTest.js'
 
-it('renders without crashing', () => {
-  const wrapper = mount(<App />);
-  const input = wrapper.find('#input1')
-  input.simulate("change", { target: { value: "potatoes" }})
-  // expect(wrapper.text()).toEqual("potatoes");
+
+describe('I.I.A.S', () => {
+  it('the user can upload recipes inputing ingredients', async () => {
+    const browser = await puppeteer.launch({
+      slowMo: 40,
+      // headless: false,
```

Comment:
Are these commented out lines needed?

Reply:
I'll add a comment that explains what they do